src/platform/site-wide/sass/merger.scss

```diff
@@ -528,6 +528,11 @@
 
   // TODO: add this into formation
   .va-sidebarnav {
+    ul.usa-accordion {
+      > li {
+        margin-bottom: -1px;
```

Comment:
need this so that way top level accordion buttons in sidenav have proper border lines